ansible/roles/common/tasks/setfacts.yml

```diff
@@ -1,11 +1,11 @@
 - name: show ansible versions
   debug: msg="You are using Ansible {{ ansible_version.full }} on {{ansible_os_family}}. The current supported version for ala-install is {{ supported_ansible_version.full }}."
 
-- name: check the ansible version
-  pause: prompt="You are not using the supported version of Ansible. The supported version is {{ supported_ansible_version.full }}. You are using {{ ansible_version.full }}. Proceed at your own risk. Press enter to continue"
-  when: supported_ansible_version.major != ansible_version.major or supported_ansible_version.minor != ansible_version.minor or supported_ansible_version.revision != ansible_version.revision
-  tags:
-    - version_check
+#- name: check the ansible version
```

Comment:
Reversing this is necessary before merging back. We have agreed widely that it is useful to check the major and minor version for the ansible product in use given there are many incompatible changes at the minor version level. The patch revision check has been removed since this branch was forked, as ansible are fairly good at only adding bug fixes at the patch revision level.